core/soc/estimator.go

```diff
@@ -51,15 +54,42 @@ func (s *Estimator) Reset() {
 	s.capacity = float64(s.vehicle.Capacity()) * 1e3  // cache to simplify debugging
 	s.virtualCapacity = s.capacity / ChargeEfficiency // initial capacity taking efficiency into account
 	s.energyPerSocStep = s.virtualCapacity / 100
+	s.minChargePower = 1000  // default 1 kW
```

Comment:
Idee:

    capacity / 50

Reply:
Ich glaube falsch wäre es nicht. Bringt aber mehr Komplexität. Ich glaube zum Schluss gewinnen wir ggf. 0.01% über die Dauer. Ich würde es aus Wartbarkeitsgründen so lassen.

Reply:
Nochmal: mir geht es um kleine Akkus. Die fixen Werte passen ausschliesslich für BEVs.

> Ich glaube zum Schluss gewinnen wir ggf. 0.01% über die Dauer.

das gilt auch für Hybride, Roller und Fahrräder? Wenn wir die Schätzung dafür deutlich verbessern könnten dann wären 2 mul/div ein fairer Preis.

Reply:
>mir geht es um kleine Akkus

Ich verstehe das Problem nicht.
Die default-Werte sind doch nur dazu da, um mathematisch den Punkt (Soc) zu ermitteln, an dem die Ladeleistung geringer wird. 
Je höher die Ladeleistung, desto früher tritt dieser Zustand ein. Das ist doch bei jedem Akku so, unabhängig von der Größe.
Es wird immer `capacity / chargePower` gerechnet. Ab dem oben genannten Punkt wird (grob gesagt) mit der halben chargePower gerechnet.
Das funktioniert auch bei 750Wh und 250W und auch bei 17kWh und 8kW.

Hier https://github.com/evcc-io/evcc/discussions/6235#discussioncomment-5014951  ist das Prinzip bildlich dargestellt.

Ich habe das für 3 Fälle (eBike, Hybrif, BEV) mal gegenübergestellt.
![image](https://user-images.githubusercontent.com/99199865/232663685-ab3c82ae-e039-4766-810e-e91d8d17e357.png)

Reply:
Auch wenn e ma Offtopic ist, aber ich liebe diese Exceltabelle. Sie spart soviel Zeit, weil man einfach schnell zeigen kann was das Ding tun würde.

Reply:
> Idee:
> 
> ```
> capacity / 50
> ```

Auch für diese relative Größe habe ich durchaus gewisse Sympathie, allein fehlt mir hier noch der theoretische oder praktische Hintergrund für diesen Zusammenhang.

Reply:
Ich habe mal ein wenig in Excel gespielt.
Vorschlag von @andig zu s.minchargePower habe ich übernommen `capacity / 50`